### ISSUE TITLE
I train the model with a private data set but it has a problem

### ISSUE THREAD

**Issue by HMCCMH** [ppengtang/pcl.pytorch#70]
```
[Dec14-09-57-00_ubuntu_step][vgg16_voc2007damperOICRv2.yaml][Step 121 / 35000]
                loss: 1.895212, lr: 0.000493 time: 0.363291, eta: 3:31:11
                loss_im_cls: 0.842698, refine_loss0: 0.001076, refine_loss1: 0.364626, refine_loss2: 0.602564
[Dec14-09-57-00_ubuntu_step][vgg16_voc2007damperOICRv2.yaml][Step 141 / 35000]
                loss: 1.790664, lr: 0.000520 time: 0.367715, eta: 3:33:38
                loss_im_cls: 0.676701, refine_loss0: 0.001097, refine_loss1: 0.436012, refine_loss2: 0.491883
[Dec14-09-57-00_ubuntu_step][vgg16_voc2007damperOICRv2.yaml][Step 161 / 35000]
                loss: 13.808914, lr: 0.000546 time: 0.374275, eta: 3:37:19
                loss_im_cls: 6.904458, refine_loss0: 1.984761, refine_loss1: 0.000000, refine_loss2: 2.014762
[Dec14-09-57-00_ubuntu_step][vgg16_voc2007damperOICRv2.yaml][Step 181 / 35000]
                loss: 15.403705, lr: 0.000573 time: 0.377689, eta: 3:39:11
                loss_im_cls: 6.904458, refine_loss0: 4.921776, refine_loss1: 0.000000, refine_loss2: 6.332109
[Dec14-09-57-00_ubuntu_step][vgg16_voc2007damperOICRv2.yaml][Step 201 / 35000]
                loss: 16.489358, lr: 0.000600 time: 0.379795, eta: 3:40:16
                loss_im_cls: 6.904458, refine_loss0: 4.383768, refine_loss1: 0.000000, refine_loss2: 4.949942
```
At the beginning, the model diverges.  At first I thought there were too many boxes provided by selective search, and I reduced the number of boxes. But it didn't work.

**Comment by RenzhiDaDa** [comment]
can you tell me how to obtain selective-search-data. this project give the prepared one

**Comment by RenzhiDaDa** [comment]
can you tell me how to obtain selective-search-data for your private dataset. this project gives the prepared one

**Comment by HMCCMH** [comment]
I think that  the images of my dataset should be normalized in the same way  as the pretrained model ? So how can I find the normalization of the pretrained model?

**Comment by HMCCMH** [comment]
> can you tell me how to obtain selective-search-data for your private dataset. this project gives the prepared one
Here is the [answer](https://github.com/ppengtang/pcl.pytorch/issues/40#issuecomment-703186389).

**Comment by HMCCMH** [comment]
I find the problem that the highest score on the first refinement provided by the WSDDN is very low(about 0.004). It means that vgg16 cannot find the right feature for my data. My object is damper.

**Comment by WangKun328** [comment]
Could you tell me how you set MAX_ITER and STEPS when training on your own private dataset? @HMCCMH

**Comment by hust-lidelong** [comment]
已收到邮件，谢谢
